[2024-12-16T19:14:57.669+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sick_leave_etl_dag.extract_data manual__2024-12-16T19:14:56.407368+00:00 [queued]>
[2024-12-16T19:14:57.676+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sick_leave_etl_dag.extract_data manual__2024-12-16T19:14:56.407368+00:00 [queued]>
[2024-12-16T19:14:57.676+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-12-16T19:14:57.688+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): extract_data> on 2024-12-16 19:14:56.407368+00:00
[2024-12-16T19:14:57.692+0000] {standard_task_runner.py:57} INFO - Started process 5403 to run task
[2024-12-16T19:14:57.694+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'sick_leave_etl_dag', 'extract_data', 'manual__2024-12-16T19:14:56.407368+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/sick_leave_dag.py', '--cfg-path', '/tmp/tmpsrbc6rsr']
[2024-12-16T19:14:57.696+0000] {standard_task_runner.py:85} INFO - Job 57: Subtask extract_data
[2024-12-16T19:14:57.712+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-16T19:14:57.746+0000] {task_command.py:415} INFO - Running <TaskInstance: sick_leave_etl_dag.extract_data manual__2024-12-16T19:14:56.407368+00:00 [running]> on host cfbf61ae1f16
[2024-12-16T19:14:57.822+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sick_leave_etl_dag' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-16T19:14:56.407368+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-16T19:14:56.407368+00:00'
[2024-12-16T19:14:57.934+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-12-16T19:14:57.944+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=sick_leave_etl_dag, task_id=extract_data, execution_date=20241216T191456, start_date=20241216T191457, end_date=20241216T191457
[2024-12-16T19:14:57.987+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-16T19:14:58.009+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
